modules/quick-start-content-guidelines.adoc

```diff
@@ -0,0 +1,165 @@
+// Module included in the following assemblies:
+//
+// * web_console/creating-quick-start-tutorials.adoc
+
+[id="quick-start-content-guidelines_{context}"]
+= Quick start content guidelines
+
+[id="quick-start-content-guidelines-card-copy_{context}"]
+== Card copy
+
+You can customize the title and description on a quick start card, but you cannot customize the status.
+
+* Keep your description to 1 to 2  sentences.
+* Start with a verb and communicate the user's goal. Correct example:
++
+----
+Create a serverless application.
+----
+
+[id="quick-start-content-guidelines-introduction_{context}"]
+== Introduction
+
+After clicking a quick start card, a side panel slides in that introduces the quick start and lists the tasks within it.
+
+* Make your introduction content clear, concise, informative, and friendly.
+* State the outcome of the quick start. A user should understand the purpose of the quick start before they begin.
+* Give action to the user, not the quick start.
+** *Correct example*:
```

Comment:
These could be `.Correct example` titles directly on the codeblock instead of sub-bullets, if you wanted.